base/range.jl

```diff
@@ -786,3 +786,5 @@ end
 
 in{T<:Integer}(x, r::Range{T}) = isinteger(x) && !isempty(r) && x>=minimum(r) && x<=maximum(r) && (mod(convert(T,x),step(r))-mod(first(r),step(r)) == 0)
 in(x::Char, r::Range{Char}) = !isempty(r) && x >= minimum(r) && x <= maximum(r) && (mod(Int(x) - Int(first(r)), step(r)) == 0)
+
+unique(x::Range) = x
```

Comment:
All I'm saying is that this is what happens with this PR, and that this sounds justified, as `Set`s are not `AbstractArray`s and e.g. cannot be indexed.